docs/guides/state_management.md

```diff
@@ -21,6 +21,71 @@ def page():
     me.text(state.val)
 ```
 
+## Use immutable default values
+
+Similar to [regular dataclasses which disallow mutable default values](https://docs.python.org/3/library/dataclasses.html#mutable-default-values), you need to avoid mutable default values such as list and dict for state classes. Using mutable default values can result in leaking state across sessions which can be a serious privacy issue.
```

Comment:
> you need to avoid mutable default values such as list and dict for state classes.

This could be accidentally misinterpreted as you can't use list/dict or any mutable data types in State classes. But I think it should be fine if they read  the whole section. And I think probably better to err on the side of caution. And I'm not sure how else you could word it in a succinct and clear way

Reply:
yeah, I could see that but with the examples, it should be clear they can use list.